core/sentry_bot.py

```
@@ -126,6 +175,105 @@ async def handle_log_analysis(message, say, context):
+            if "handle sentry" in query_text.lower():
+                return await handle_sentry_issue(event, say, context)
+            # Check for summarization keywords
+            if any(keyword in query_text.lower() for keyword in ['summary', 'summarise', 'summarize']):
+                url_match = re.search(r"<https?://[\w.-]+\.slack\.com/archives/(\w+)/p(\d+)>", query_text)
+                if url_match:
+                    # URL summarization
+                    summarize_channel_id = url_match.group(1)
+                    ts_string = url_match.group(2)
+                    summarize_thread_ts = f"{ts_string[:-6]}.{ts_string[-6:]}"
+                    
+                    await client.chat_postMessage(channel=channel, thread_ts=thread_ts, text=f"Sure, I'll summarize the linked thread for you. One moment... 🧐")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)